javascripts/govuk/show-hide-content.js

```diff
@@ -80,22 +79,23 @@
     // Handle radio show/hide
     function handleRadioContent ($control, $content) {
 
-      // All radios in this group
-      var selector = selectors.radio + '[name=' + escapeElementName($control.attr('name')) + ']'
+      // All radios in this group which control content
+      var selector = selectors.radio + '[name=' + escapeElementName($control.attr('name')) + '][aria-controls]'
```

Comment:
Hi @tombye, @gemmaleigh.

This has a side effect unfortunately… Now *only* controlling radios can be used to show/hide content.

Picture a yes or no question and answer form, for example "Do you have a partner"?

Previously the "Yes" radio (controlling) could toggle the content open with further content, and the "No" radio in the same `name` group (not controlling) could toggle the content closed again.

With this latest change, how do you close it once it's open?